ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -376,6 +416,8 @@ public void onCompletion(Long result, Exception exception) {
         routerCallback.onPollReady();
       }
     };
+    // the index of the next chunk that is to be written out to the asyncWritableChannel.
+    private int indexOfNextChunkToWriteOut = 0;
```

Comment:
It was auto moved by intellij code formatting tool. Let me know if you want me to move it back.

Reply:
Ah, it doesn't do that for me since I don't have rearrange code check. Just keep it in whatever place you feel is best or most logical, I don't think we have any designated style for this